frontend/encore/installation.rst

```diff
@@ -1,46 +1,28 @@
-Encore Installation
+Installating Encore
 ===================
 
 First, make sure you `install Node.js`_ and also the `Yarn package manager`_.
 
-Then, install Encore into your project with Yarn:
+Option A) If you are using Symfony Flex
```

Comment:
Why not do like you did with the non-flex installation page (with a tip at the top)? It's kind of ugly to have options on the "main" page about Encore installation as 99% of the time, people will have Flex.